biogeophys/FatesHydroWTFMod.F90

```diff
@@ -304,6 +304,7 @@ end subroutine set_wrf_param_base
   function get_thsat_base(this) result(th_sat)
     class(wrf_type)     :: this
     real(r8)            :: th_sat
+    th_sat = 0._r8
```

Comment:
Build error:
```
"/autofs/nccsopen-svm1_home/azamat/repos/E3SM/components/elm/src/external_models/fates/biogeophys/FatesHydroWTFMod.F90", 1513-083 (E) Internal or module function th_sat was not set within the function.
1501-511  Compilation failed for file FatesHydroWTFMod.F90.
```

Reply:
thanks @amametjanov , this seems pretty straight-forward